bigbone-rx/src/main/kotlin/social/bigbone/rx/RxOAuthMethods.kt

```diff
@@ -15,32 +15,49 @@ import social.bigbone.rx.extensions.onErrorIfNotDisposed
 class RxOAuthMethods(client: MastodonClient) {
     private val oauth = OAuthMethods(client)
 
-    fun getAccessTokenWithAuthorizationCodeGrant(
+    fun getClientAccessToken(
```

Comment:
You have implemented the same method in one of your other pull requests. I'll have a look and see which and how we merge this.

Reply:
Done.